internal/versions/versions.go

```diff
@@ -76,112 +81,192 @@ func (h *handler) ListSupportedOpenshiftVersions(ctx context.Context, params ope
 	return operations.NewListSupportedOpenshiftVersionsOK().WithPayload(h.openshiftVersions)
 }
 
-func (h *handler) GetReleaseImage(openshiftVersion string) (pullSpec string, err error) {
+func (h *handler) GetReleaseImage(openshiftVersion, cpuArchitecture string) (pullSpec string, err error) {
 	versionKey, err := h.GetKey(openshiftVersion)
 	if err != nil {
 		return "", err
 	}
-	if !h.IsOpenshiftVersionSupported(versionKey) {
-		return "", errors.Errorf("No release image for unsupported openshift version %s", versionKey)
+	if !h.IsOpenshiftVersionSupported(versionKey, cpuArchitecture) {
+		return "", errors.Errorf("No release image for unsupported openshift version %s, CPU architecture %s", versionKey, cpuArchitecture)
+	}
+
+	if h.openshiftVersions[versionKey].Images != nil {
+		if images, ok := h.openshiftVersions[versionKey].Images[cpuArchitecture]; ok {
+			return *images.ReleaseImage, nil
+		}
+		return "", errors.Errorf("Release image was missing for openshift version %s, CPU architecture %s", versionKey, cpuArchitecture)
 	}
 
 	if h.openshiftVersions[versionKey].ReleaseImage == nil {
-		return "", errors.Errorf("Release image was missing for openshift version %s", versionKey)
+		return "", errors.Errorf("Release image was missing for openshift version %s, CPU architecture %s", versionKey, cpuArchitecture)
 	}
```

Comment:
These two logs look the same. They should be different so it would be easier to understand what's the failure.